### ISSUE TITLE
Voiceover improvements

### ISSUE THREAD

**Issue by NicoHood** [nims11/IPod-Shuffle-4g#56]
There is a fork with some improvements to the voiceover. We should have a look and maybe integrate those fixes?
https://github.com/thomas-hori/IPod-Shuffle-4g/commit/0a72c1102d25101749279f6071ebacdd7f1ca7eb
